facebook-for-woocommerce.php

```diff
@@ -12,7 +12,7 @@
  * Author: Facebook
  * Author URI: https://www.facebook.com/
  * Version: 3.2.5
- * Requires at least: 5.6
+ * Requires at least: 7.4
```

Comment:
This is the WordPress version. I am setting the same as the one used in WooCommerce..
```suggestion
 * Requires at least: 6.4
 * Requires PHP: 7.4
```